WalletWasabi.Gui/Styles/Styles.xaml

```diff
@@ -94,6 +94,21 @@
       </Template>
     </Setter>
   </Style>
+  <Style Selector="Button.greenButton">
+    <Setter Property="Foreground" Value="White" />
+    <Setter Property="Background" Value="DarkGreen" />
+  </Style>
+  <Style Selector="Button.greenButton > TextBlock">
+    <Setter Property="Foreground" Value="White" />
+    <Setter Property="Margin" Value="7" />
```

Comment:
How about lowering this to 2 or 3, currently the buttons look thick IMO.

Master:
![Capture3-](https://user-images.githubusercontent.com/52379387/92529281-84df5b00-f22a-11ea-995c-ebf4466d5f0d.PNG)

This PR:
![Capture3](https://user-images.githubusercontent.com/52379387/92529294-8c9eff80-f22a-11ea-9a9b-ef5182b1393a.PNG)

This PR with value = 2:
![aaza](https://user-images.githubusercontent.com/52379387/92529323-96c0fe00-f22a-11ea-90f2-839acc2e61d0.PNG)